api/src/main/java/gov/cms/ab2d/api/controller/HealthAPI.java

```diff
@@ -22,6 +23,8 @@
     @Autowired
     private LogManager eventLogger;
 
+    // Add exceptions for testing and prod site
+    @CrossOrigin(origins = {"http://127.0.0.1:4000", "https://ab2d.cms.gov", "http://ab2d.cms.gov"})
```

Comment:
Is this really wise to add this to production code or should this just be something documented for testing?

Reply:
CrossOrigin is inherent to the browser, it's just saying that it will allow all origins to access it from a browser. It was already accessible from anywhere e.g. if someone made postman calls or used a client library to make API calls to it.